### PR TITLE
Add Quick Select (Selection)

### PR DESCRIPTION
**Pull request by thakreyn**
Initialized a new directory in 'en' for Selection Algorithms.
Created a first entry in the Selection Algorithms directory in the form of  Quick Select.md, following the standard format.

### CODE REVIEW

File: en/Selection Algorithms/Quick Select.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Else, check if 'k' is greater or smaller and choose a new pivot in that range.
+- Repeat till we get the kth element at kth position
+
+### Time Complexity
+
+`O(n^2)` Worst Case Performance
+
+`O(n)` Best-case Performance
+
+`O(n)` Average Performance
```
Comment (Panquesito7) [inline en/Selection Algorithms/Quick Select.md:22]:
```suggestion
- `O(n^2)` Worst-Case Performance

- `O(n)` Best-case Performance

- `O(n)` Average Performance
```

**Review by Panquesito7 — COMMENTED** [review]
👍 Good work!

**Review by vbrazo — APPROVED** [review]
LGTM

**Comment by Panquesito7** [comment]
@appgurueu, would you like to review this PR, please? Thanks. 🙂

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 👍 🎉

**Review by appgurueu — APPROVED** [review]
Fine, LGTM

**Pull request merged**